whitepaper/whitepaper.md

```diff
@@ -706,19 +706,25 @@ Once an API call to Hyperledger Cactus framework is requested by a User, Busines
+- **Command API Server (cmd-api-server)**: The server accepts a request from an End-user Application, and return a response depending on the status of the targeted trade. Trade ID will be assigned when a new trade is accepted.
+- **End-user Application**: The entity submits API calls to request a trade, which invokes a set of transactions on Ledger by the Business Logic Plugin.
+- **Ledger Event Listener**: The standard interface to handle various kinds of events(LedgerEvent) regarding asynchronous Ledger operations. The LedgerEvent will be notified to appropriate business logic to handle it.
 - **Ledger Plugin**: The entity communicates Business Logic Plugin with each ledger.  The entity is composed by a validator and a verifier as follows. The entity(s) is(are) chosen from multiple plugins on configuration.
+- **Service Provider Application**: The entity submits API calls to control the cmd-api-server when it is enabling/disabling Ledger plugins, or shutting down the server. Additional commands may be available on Admin API since **Server controller** is implementation-dependent.
 - **Validator**: The entity monitors transaction records of Ledger operation, and it determines the result(success, failed, timeouted) from the transaction records.
 Validator ensure the determined result with attaching digital signature with "Validator key" which can be verified by "Verifier".
-- **Verifier**: The entity accepts only sucussfully verified operation results by verifying the digital signature of the validator. Note that "Validator" is apart from "Verifier" over a bi-directional channel.
-- **Cactus Routing Interface**: The entity is a routing service between "Business Logic Plugin" and  "Ledger Plugin(s)". The entity is also a routing service between Business Logic Plugin and API calls from "Application user(s)".
-- **Ledger-n**: DLT platforms(e.g. Ethereum, Quorum, Hyperledger Fabric, ...)
+- **Validator Server**: The server accepts a connection from Verifier, and it provides Validator API, which can be used for controlling and monitoring Ledger behind it. The LedgerConnector will be implemented for interacting with the Ledger nodes.
+- **Verifier**: The entity accepts only sucussfully verified operation results by verifying the digital signature of the validator. Verifier will be instantiated by calling the VerifierFactory#create method with associated with the Validator to connect. Each Verifier may be temporarily enabled or disabled. Note that "Validator" is apart from "Verifier" over a bi-directional channel.
+- **Verifier Registroy**: The information about active Verifier. The VerifierFactory uses this information to instantiate Verifier for the Business Logic Plugin. 
```

Comment:
@sfuji822 

> Verifier Registroy

Typo? https://en.wikipedia.org/wiki/Registry

> The information about active Verifier.

Is it implied there that there's only one verifier active or could it be multiple active verifiers simultaneously ?

Reply:
I will fix the typo 'Registry'. 

> Is it implied there that there's only one verifier active or could it be multiple active verifiers simultaneously ?

I thought that, but we may clarify if Verifier(s) to be multiple instances or not. 
Our prototyped code have instanced interface to Validator once it was created by ValidatorFactory.
I would like to hear your opinion about this. > @petermetz

Reply:
> I thought that, 

@sfuji822 Sorry, wasn't sure which one of the two you meant by `that`. Do you mean a) `there's only one verifier active` or b) `could it be multiple active verifiers simultaneously`